Add info of which extrinsics can be submitted in each OpenGov Track (#6680)
w3f/polkadot-wiki: docs/learn/learn-polkadot-opengov-origins.md

    For every referendum in each of these tracks, the Polkadot-JS UI displays interactive graphs of the support and approval.
    ![UI Support and Approval](./../assets/governance/support-approval-polkadot-js-ui.png)

## Extrinsics and Tracks

> The following table outlines which extrinsics can be placed on which governance tracks:

Reviewer: ```suggestion
The following table outlines which extrinsics can be submitted on which governance tracks.
```

    For every referendum in each of these tracks, the Polkadot-JS UI displays interactive graphs of the support and approval.
    ![UI Support and Approval](./../assets/governance/support-approval-polkadot-js-ui.png)

## Extrinsics and Tracks

!!!note

>     The following extrinsics must only be sent either with the `Root` or OpenGov origins, which are detailed in the table.

Reviewer: ```suggestion
    You must submit the extrinsics in the table below using either the Root or a compatible OpenGov origin.
```

> The following table outlines which extrinsics can be submitted on which governance tracks.

Reviewer: ```suggestion
```
